UnicornEngine/video/include/unicorn/video/vulkan/Renderer.hpp

```diff
@@ -26,56 +21,83 @@ class Window;
 
 namespace video
 {
+namespace vulkan
+{
+/**
+ * @brief Struct for easy check of required queue family indices
+ */
 struct QueueFamilyIndices
 {
-    int graphicsFamily = -1;
-    int presentFamily = -1;
-
-    bool IsComplete() const
-    {
-        return graphicsFamily >= 0 && presentFamily >= 0;
-    }
+    int32_t graphicsFamily = -1;
+    int32_t presentFamily = -1;
+
+    /**
+     * @brief Check if all needed family indices are exists.
+     * @return true if graphics and present families exists and false if not
+     */
+    bool IsComplete() const;
 };
 
+/**
+ * @brief Details of swapchain for creating it.
+ */
 struct SwapChainSupportDetails
 {
     vk::SurfaceCapabilitiesKHR capabilities;
     std::vector<vk::SurfaceFormatKHR> formats;
     std::vector<vk::PresentModeKHR> presentModes;
 };
 
-/** @brief  Vulkan renderer class
- *
- *  Initializes and controls Vulkan API
+/**
+ * @brief Struct for easy filling and sending to shader
```

Comment:
`UniformCameraData` has nothing to do with shaders, it holds camera information. Do not describe implementation details in documentation